Added integration docs (#211)
maykinmedia/archiefvernietigingscomponent: doc/manual/faq.rst

.. _faq:

Frequently asked questions
==========================

> What is exactly is deleted by the application?

Reviewer: ```suggestion
What exactly is deleted by the application?
```

as present in for example `Open Zaak`_. If the application has the proper 
authorizations for the APIs and a destruction list with cases (zaken) is 
approved, the application will delete:

* The cases in the `Zaken API`_ and all related objects within the same API, 
  including the audit trail.
* All documents (documenten) in the `Documenten API`_, related to the deleted 
  case if the document is not related to any other case. All related objects to 
  the document within the same API are also deleted. If the document is related 
  to another case, only the relation to the deleted case, is deleted.
* All decisions (besluiten) in the `Besluiten API`_, related to the deleted 
  case. All related objects to the decision within the same API are also 
  deleted.

Are things really deleted?
--------------------------

Yes! Deleting or destroying cases (zaken) will remove the record from the 
database. It's not just a flag that is set on the deleted case.

I accidentally deleted a case, can I get it back?
-------------------------------------------------

No! Unless you have a backup of the data, you cannot undo a deletion or get the
data back.

> Is related data, not part of the API's voor Zaakgericht Werken, deleted?

Reviewer: ```suggestion
Is related data, not part of the `API's voor Zaakgericht Werken`_, deleted?
```

------------------------------------------------------------------------

The application destroys cases and related documents and decisions in the 
`API's voor Zaakgericht Werken`_. The application has no direct control over

> data that is stored in other applications, registers, API's or components.

Reviewer: ```suggestion
data that is stored in other applications, registers, APIs or components.
```

> It's the responsibility of the recordmanager or archivist to make sure that 

Reviewer: ```suggestion
It's the responsibility of the record manager or archivist to make sure that 
```
